src/Units/LanderModel.java

```diff
@@ -0,0 +1,32 @@
+package Units;
+
+import java.util.Vector;
+
+import Engine.GameAction.ActionType;
+import Terrain.Environment.Terrains;
+import Units.MoveTypes.FloatLight;
+import Units.MoveTypes.MoveType;
+
+public class LanderModel extends UnitModel
+{
+
+  private static final MoveType moveType = new FloatLight();
+  // TODO: limit the valid locations to unload
```

Comment:
Can you provide a little more explanation? I'm currently not sure what needs to be done. I can infer that maybe we would currently get an "Unload" prompt when there are no valid drop locations, but a little extra text here would go a long way towards making sure this actually gets done. A descriptive issue would go even farther.